test/apps/Makefile.am

```diff
@@ -45,7 +45,7 @@ bin_PROGRAMS = $(check_PROGRAMS)
 AM_CPPFLAGS = -I$(top_srcdir)/test/include
 LDADD =
 else
-AM_CPPFLAGS = -I$(top_builddir)/mpp -I$(top_srcdir)/mpp -I$(top_srcdir)/test/include
+AM_CPPFLAGS = -I$(top_builddir)/mpp -I$(top_srcdir)/mpp -I../include
```

Comment:
Just to make sure, we dont have to do the same change for `mpp` dir?

Reply:
We do not need to make the change in `mpp`, because building `tests-sos` as a submodule within SOS will use `top_builddir` and `top_srcdir` with respect to _SOS_, which I think is what we want.

However, your question made me realize I forgot this change in `shmemx` and `spec-example` directories, which didn't produce any error because we are not including internal headers there.  I fixed that in https://github.com/openshmem-org/tests-sos/pull/32/commits/e1b2fad8ededcf36e3a3f685fa7ed229ddb0fefa